src/components/container/SeverityCalendar/SeverityCalendar.tsx

```diff
@@ -0,0 +1,149 @@
+import { useContext, useState } from "react";
+import { Guid, SurveyAnswer, SurveyAnswersQuery } from "@careevolution/mydatahelps-js";
+import React from "react";
+import surveyResults from "../../../helpers/get-survey-answers";
+import Calendar from "../../presentational/Calendar/Calendar";
+import { parseISO, startOfDay, startOfMonth } from "date-fns";
+import { CalendarDay, CalendarDayStateConfiguration, DateRangeContext, LoadingIndicator } from "../../presentational";
+import { previewSeverityData } from "./SeverityCalendar.previewdata";
+import { useInitializeView } from "../../../helpers/Initialization";
+export type SeverityCalendarPreviewState = "WithData" | "NoData" | "Loading" | "Live";
```

Comment:
You shouldn't have a "Live" preview state - not having a preview state defined is "Live" (it is the default use of the component)